modules/5_install/install.tf

```diff
@@ -448,3 +448,13 @@ resource "null_resource" "csi_driver_config" {
     ]
   }
 }
+
+resource "ibm_pi_operations" "pi-my-ins" {
+       depends_on = [null_resource.config, null_resource.setup_snat, null_resource.configure_public_vip, null_resource.external_services, null_resource.pre_install, null_resource.install, null_resource.powervs_config, null_resource.upgrade, null_resource.csi_driver_config ]
+        count      =  var.fips_compliant  ? 1 : 0
```

Comment:
count should also include number of bastion nodes in case of bastion in HA mode.